src/components/Header/__snapshots__/Header.test.js.snap

```diff
@@ -0,0 +1,115 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
the snapshot doesn't seem like a proper representation of the header components. Will ignore this for now and will update when we setup the test properly.